crates/hyperswitch_connectors/src/connectors/nexixpay/transformers.rs

```diff
@@ -55,26 +305,46 @@ impl TryFrom<&NexixpayRouterData<&PaymentsAuthorizeRouterData>> for NexixpayPaym
         item: &NexixpayRouterData<&PaymentsAuthorizeRouterData>,
     ) -> Result<Self, Self::Error> {
         match item.router_data.request.payment_method_data.clone() {
-            PaymentMethodData::Card(req_card) => {
+            PaymentMethodData::Card(ref req_card) => {
                 let card = NexixpayCard {
-                    number: req_card.card_number,
-                    expiry_month: req_card.card_exp_month,
-                    expiry_year: req_card.card_exp_year,
-                    cvc: req_card.card_cvc,
-                    complete: item.router_data.request.is_auto_capture()?,
+                    pan: req_card.card_number.clone(),
+                    expiry_date: req_card.get_expiry_date_as_mmyy()?,
+                    cvv: Some(req_card.card_cvc.clone()),
                 };
-                Ok(Self {
+                let billing_address = Address {
+                    name: item.router_data.get_optional_billing_full_name().clone(),
+                    street: item.router_data.get_optional_billing_line1(),
+                    additional_info: None,
+                    city: item.router_data.get_optional_billing_city(),
+                    post_code: item.router_data.get_optional_billing_zip(),
+                    province: None,
+                    country: item.router_data.get_optional_billing_country(),
+                };
+                let customer_info = CustomerInfo {
+                    card_holder_name: item.router_data.get_optional_billing_full_name().clone(),
+                    billing_address: billing_address.clone(),
+                    shipping_address: billing_address,
+                };
+                let order = Order {
+                    order_id: item.router_data.connector_request_reference_id.clone(),
                     amount: item.amount.clone(),
+                    currency: item.router_data.request.currency,
+                    description: None,
+                    custom_field: None,
+                    customer_info: Some(customer_info),
+                };
+                Ok(Self {
+                    order,
                     card,
+                    recurrence: None,
+                    exemptions: None,
                 })
             }
             _ => Err(errors::ConnectorError::NotImplemented("Payment methods".to_string()).into()),
```

Comment:
avoid defaults(_=>)
```suggestion
            _ => Err(errors::ConnectorError::NotImplemented("Payment method".to_string()).into()),
```

Reply:
should i mention all payment methods instead of '_' ?

Reply:
sure you can mention all the payment methods and use this function as well
`Err(errors::ConnectorError::NotImplemented(
                    utils::get_unimplemented_payment_method_error_message("Nexixpay"),
                ))?`

Reply:
Please cater to this comment and dont use `_`